tests/ContaoCoreBundleTest.php

```diff
@@ -60,16 +60,23 @@ public function testAddsTheCompilerPaths()
         $bundle->build($container);
 
         $classes = [];
+        $beforeRemovingClasses = [];
 
         foreach ($container->getCompilerPassConfig()->getPasses() as $pass) {
             $reflection = new \ReflectionClass($pass);
             $classes[] = $reflection->getName();
         }
 
+        foreach ($container->getCompilerPassConfig()->getBeforeRemovingPasses() as $pass) {
+            $reflection = new \ReflectionClass($pass);
+            $beforeRemovingClasses[] = $reflection->getName();
+        }
+
         $this->assertContains(AddPackagesPass::class, $classes);
         $this->assertContains(AddSessionBagsPass::class, $classes);
         $this->assertContains(AddResourcesPathsPass::class, $classes);
         $this->assertContains(AddImagineClassPass::class, $classes);
         $this->assertContains(DoctrineMigrationsPass::class, $classes);
```

Comment:
Shouldn't we delete this line?

Reply:
It should be in both arrays 😄

Reply:
We probably should change `getPasses()` to `getBeforeOptimizationPasses()` to be more explicit then.